api-reference/_hidden/dxHtmlEditorTableResizing/minColumnWidth.md

```diff
@@ -5,7 +5,8 @@ default: 40
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+##### shortDescription
+The minimum column width.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+When the content width exceeds this property's value and it is impossible to move the content to a new line, the actual column width equals the content width. 
```

Comment:
Does this sentence describe the case when a user wants to shrink a column down to the minColumnWidth value, but the column won't let him?

Reply:
Yes, exactly.

Reply:
Then it's unclear that you mean this case. Please rewrite.